compatibility_lib/compatibility_lib/configs.py

```diff
@@ -95,6 +104,29 @@ def _format_url(repo_name, setuppy_path=''):
     'grpc-google-iam-v1',
     'grpcio',
     'opencensus',
+    'opencensus-correlation',
+    'opencensus-ext-azure',
+    'opencensus-ext-dbapi',
+    'opencensus-ext-django',
+    'opencensus-ext-flask',
+    'opencensus-ext-gevent',
+    'opencensus-ext-google',
```

Comment:
I didn't find any package named this on PyPI.

Reply:
this was a typo as it should have been `opencensus-ext-google-cloud-clientlibs` - fixed this.